### ISSUE TITLE
[UI] Load risk view before rendering

### ISSUE THREAD

**Issue by tens0rfl0w** [corona-warn-app/cwa-app-ios#770]
## Avoid duplicates

* [x] This issue has not already been raised before
* [x] If you are proposing a new feature, please do so in [CWA-Wishlist](https://github.com/corona-warn-app/cwa-wishlist)

## Current Implementation
If you launch the app, the risk view cell initially renders as grey and with the status „unknown“ and then loads the actual risk and changes the appearance 
## Suggested Enhancement
Load the actual risk status before rendering the risk view cell, so the first render is the correct one.
## Expected Benefits
Prettier UI is always a good thing😄

---
Internal Tracking ID: EXPOSUREAPP-1946

**Comment by tens0rfl0w** [comment]
I guess this is fully fixed after the merging of the mentioned PR @SebastianWild ?🙂

If yes, feel free to close this one.

**Comment by taslior** [comment]
This issue does still exist in release 1.1.1, at least on my iPhone XR running 13.6. 
After updating the app today to 1.1.1 it took several seconds till the risk view cell was actually several seconds in status grey until it finally switches to green. Subsequent starts of the CWA show a short flicker as before between grey until it settles to green

**Comment by tens0rfl0w** [comment]
@taslior Please see the changelog to have an overview of all PRs that were included.

**Comment by taslior** [comment]
Ok. According to https://github.com/corona-warn-app/cwa-app-ios/releases [ Home Screen Risk Cell Update on App start + Risk Calc Refinements #874](https://github.com/corona-warn-app/cwa-app-ios/pull/874) should have been included in 1.0.5 already. 
As stated above, in 1.1.1. I still can see a brief "Risikoermittlung gestoppt" until I get the final status. 

I am not used to github, should I comment on #874, reopen #874 or just comment on this issues, which describes the same behavior and is still in status open or create a new issue?

**Comment by Skibbe69** [comment]
For me this behavoir starts with the latest update on CWA 1.1.2. Never seen this before. 
Technical Info:
iPhone 11 Pro
iOS 13.6

**Comment by GPclips** [comment]
Hello @tens0rfl0w and community,

thank you for reporting the issue.
I will follow up with the dev team and we will get back to you as soon as possible.

Thanks,
LMM

Corona-Warn-App Open Source Team

**Comment by tens0rfl0w** [comment]
Some additional input:
The above mentioned PR from some months ago changed the behavior of the risk status cell when it initializes on app startup. For me this is still not the best approach, as it still produces a short flicker when switching from the "Exposure logging stopped" status to the actual risk status. Also this transition seems to be heavily influenced by general device workload, as sometimes the first render already shows the actual risk status and sometimes transitions quite slowly. (Or more affected by memory caching rules after force closing the app)

My enhancement suggestion would be to initialize and show the main app screen only after initialization of the app backend has finished and the current tracking and exposure status has been determined. (Excluded from this is of course the data fetch on app start up as this could take several seconds to complete and has a fully working animation implemented)

**Comment by Ein-Tim** [comment]
This Issue is still here in CWA Version 1.5.0/1.5.1, for more Information see: #1400

**Comment by Ein-Tim** [comment]
This seems to be fixed since version 1.7.1 in general. 

There are still some edge cases (heavy device load, etc.) where the App shows the "Risiko-Ermittlung gestoppt" screen until the check has been done, but normally the ["Check is running"](https://user-images.githubusercontent.com/66167854/99144346-453b5200-2665-11eb-972c-81af71aefce5.jpeg) screen is shown.

**Comment by dsarkar** [comment]
@tens0rfl0w Any objections in closing this issue? Thanks for the feedback.

**Comment by tens0rfl0w** [comment]
The current implementation seems to work how it should. Closed👍🏼

**Comment by dsarkar** [comment]
@tens0rfl0w Thanks for feedback!
